packages/@react-spectrum/s2/stories/CoachMark.stories.tsx

```diff
@@ -0,0 +1,162 @@
+      </Footer>
+    </CoachMark>
+  </CoachMarkTrigger>
+);
+
+function CoachMarkBase({step, currentStep, totalSteps, description = '', skipTour, restartTour, advanceStep, previousStep, hasPressAction = false, placement = 'right top' as Placement}) {
+  const onAction = actionKey => {
+    if (actionKey === 'skip') {
+      skipTour();
+    } else if (actionKey === 'restart') {
+      restartTour();
+    }
+  };
+  return (
+    <CoachMark placement={placement}>
+      <CardPreview>
+        <Image src={new URL('assets/preview.png', import.meta.url).toString()} />
+      </CardPreview>
+      <Content>
+        <Text slot="title">Coach mark title</Text>
+        <ActionMenu onAction={onAction}>
+          <MenuItem id="skip">Skip tour</MenuItem>
+          {step !== 1 && <MenuItem id="restart">Restart tour</MenuItem>}
+        </ActionMenu>
+        <Keyboard>Command + B</Keyboard>
+        <Text slot="description">{description || 'This is the description'}</Text>
+      </Content>
+      <Footer>
+        <Text slot="steps">{currentStep} of {totalSteps}</Text>
+        {step !== 1 && <Button fillStyle="outline" variant="secondary" onPress={previousStep}>Previous</Button>}
+        {step !== totalSteps && !hasPressAction && <Button variant="primary" onPress={advanceStep}>Next</Button>}
+        {step === totalSteps && !hasPressAction && <Button variant="primary" onPress={advanceStep}>Finish</Button>}
+      </Footer>
+    </CoachMark>
+  );
+}
+
+export const TourExample = () => {
+  let {coachMarkTriggerProps, ...otherTourProps} = useTour({});
+  return (<div className={style({display: 'flex'})}>
+    <div className={style({display: 'flex', flexDirection: 'column', alignItems: 'start', gap: 16})}>
+      <CoachMarkTrigger {...coachMarkTriggerProps(1)}>
+        <ActionButton >
+          <Filter />
+        </ActionButton>
+        <CoachMarkBase step={1} {...otherTourProps} description="This is the action button." />
+      </CoachMarkTrigger>
+
+      <CoachMarkTrigger {...coachMarkTriggerProps(3)}>
+        <Slider label="Horizontal position" labelPosition="top" />
+        <CoachMarkBase step={3} {...otherTourProps} description="Adjust this slider to change the value." />
+      </CoachMarkTrigger>
+
+      <CoachMarkTrigger {...coachMarkTriggerProps(4)}>
+        <Checkbox>Sync with CC</Checkbox>
+        <CoachMarkBase step={4} {...otherTourProps} description="Check this box to sync with Creative Cloud" />
+      </CoachMarkTrigger>
+    </div>
+    <div style={{width: 500}} />
+    <div className={style({display: 'flex', flexDirection: 'column', gap: 8, alignItems: 'end'})}>
+      <CoachMarkTrigger {...coachMarkTriggerProps(2)}>
+        <Button onPress={otherTourProps.advanceStep}>
+          Apply
+        </Button>
+        <CoachMarkBase
+          step={2}
+          placement="left top"
+          {...otherTourProps}
+          description="This is the Apply button. Press it to continue the tour."
+          hasPressAction />
```

Comment:
This is really strange IMO. I guess it's overriding the press events on the button? Does the button also do its normal action?